lib/github.js

```diff
@@ -5,7 +5,6 @@ var spawn = require('child_process').spawn
 var github = exports;
 
 function Organization(options) {
-    this.BASE_URI = 'https://api.github.com/orgs/';
```

Comment:
I've never considered GitHub Enterprise, but it makes sense to support it somehow. Your solution is good, but I think we don't need a function for that. You could just change this line to:
```js
this.BASE_URI = process.env.ENTERPRISE_DOMAIN || 'https://api.github.com/orgs/'
```
What to you think?